dradis-pentera.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'dradis-plugins', '~> 4.0'
 
```

Comment:
Any reason for not including bundler and rake?
```
 spec.add_development_dependency 'bundler'
 spec.add_development_dependency 'rake'
```

Reply:
Do we need these? `rake build` seems to work as normal and running the rake tasks in dradis works as expected.